File: PPA/DailyFlash/August/20August/Program1.c


#include<stdio.h>
#include<math.h>

void main() {

	int side1, side2, hypo=0,s=0;

	printf("Enter the two side of triangle and hypotenuse: ");
	scanf("%d%d%d",&side1,&side2,&hypo);
	
	s=side1*side1+ side2*side2;
	
	if(sqrt(s) == hypo)
		printf("Triangle satisfies the Pythagorean Theorem\n");
	else
		printf("Triangle does not satisfies the Pythagorean Theorem\n");
}


	

File: Java/DailyFlash/August/20August/Program1.java

import java.util.Scanner;
import java.lang.Math.*;
class Pythagoras {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int side1, side2, hyp, s;

		System.out.print("Enter the side1, side2 and hypotenuse of triangle: ");

		side1=sc.nextInt();
		side2=sc.nextInt();
		hyp=sc.nextInt();

		s=side1*side1+ side2*side2;

		if(Math.sqrt(s) ==hyp)
			System.out.println("Triangle satisfies the Pythagorean Theorem");
		else
			System.out.println("Triangle does not satisfies the Pythagorean Theorem");
	}
}



